editor/export/editor_export_platform_apple_embedded.h

```diff
@@ -288,7 +288,7 @@ class EditorExportPlatformAppleEmbedded : public EditorExportPlatform {
 			Vector<String> plugins_filenames = list_plugin_config_files(plugins_dir, true);
 
 			if (!plugins_filenames.is_empty()) {
-				Ref<ConfigFile> config_file;
+				Ref<ConfigFile> config_file = memnew(ConfigFile);
```

Comment:
Nitpick, but we usually favor this syntax:
```suggestion
				Ref<ConfigFile> config_file;
				config_file.instantiate();
```